build/Dockerfile.ubuntu

```diff
@@ -57,18 +56,28 @@ RUN apt-get install -y apt-utils \
     curl \
     connect-proxy \
     coreutils \
-    sudo && \
-    apt-get -y clean all
+    vim \
+    sudo
 
-# Installing all PYTHON packages
-RUN pip install --upgrade pip && \
-    pip install grpcio \
-    ovspy \
+RUN if [ "$OS_VERSION" = "18.04" ] ; then \
```

Comment:
This is the only place where OS_VERSION is being used, so is it an option to compare with "$BASE_IMAGE" = "ubuntu:18.04" ? That will give us one less configuration argument to think about. 🤔

Reply:
good catch!!  Yes, we can reuse BASE_IMG to make the comparison

Reply:
I really like the idea of ipdk install [OS options]